player/client.c

```diff
@@ -1289,6 +1295,11 @@ static void setproperty_fn(void *arg)
     struct mpv_node tmp;
     if (req->format == MPV_FORMAT_NODE) {
         node = req->data;
+    } else if (req->format == MPV_FORMAT_FLAG) {
+        // Convert int to bool.
+        tmp.format = MPV_FORMAT_BOOL;
+        tmp.u.bool_ = *(bool *)req->data;
```

Comment:
Reading int as bool.

Reply:
Oh right, you can't just directly cast this. This is just me being dumb.